choclo/prism/_gravity.py

```diff
@@ -727,21 +917,53 @@ def gravity_nn(easting, northing, upward, prism, density):
     # For the g_nn this are edges perpendicular to the northing direction
     # (parallel to easting and upward)
     if (
-        is_point_on_easting_edge(easting, northing, upward, prism)
-        or is_point_on_upward_edge(easting, northing, upward, prism)
+        is_point_on_easting_edge(easting, northing, upward, prism_west, prism_east, prism_south, prism_north, prism_bottom, prism_top)
```

Comment:
I think we should remove the `# fmt: skip` comment and let black reformat it to:
```python
    if is_point_on_easting_edge(
        easting,
        northing,
        upward,
        prism_west,
        prism_east,
        prism_south,
        prism_north,
        prism_bottom,
        prism_top,
    ) or is_point_on_upward_edge(
        easting,
        northing,
        upward,
        prism_west,
        prism_east,
        prism_south,
        prism_north,
        prism_bottom,
        prism_top,
    ):
```